frontend/src/App.tsx

```diff
@@ -23,6 +23,7 @@ export class App extends React.Component {
           <Route path="/legal-disclosure" component={Legal} />
           <Route path="/privacy-policy" component={Privacy} />
           <Route path="/tracks/product-management" component={TrackPM}></Route>
+          <Route path="/tracks/product-management" component={TrackPM}></Route>
```

Comment:
As mentioned on the phone, we need unique urls here.
This is an example for Software engineering.
```suggestion
          <Route path="/tracks/software-engineering" component={TrackSE}></Route>
```

(Don't forget to import `TrackSE` and the others at the top of the file. 🤗 )